### PR TITLE
section about Mocking in README + minimal emacs mode

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
as commit name says :)

### CODE REVIEW

**Comment by AdrieanKhisbe** [comment]
*"On my launch"*, I added other features, introduce some new functionnality tha would deserve a 0.2.

I haven't complete all I have in mind at the moment, and will do a separate pull request once done. (you can have a look on the **dev** branch https://github.com/AdrieanKhisbe/shpec/tree/dev

(in a nutchell; zsh support, after/before/run/load)

File: README.md [hunk 1]
```diff
@@ -30,6 +30,13 @@ find . -name "*_shpec.sh" | entr shpec
 [shpec's own tests](https://github.com/rylnd/shpec/tree/master/shpec/shpec_shpec.sh)
 are a great place to start. More detailed examples to come.
 
+### Mocking
```
Comment (rylnd) [inline README.md]: Can we move this section after 'Matchers' and call it "Command Stubbing" ? We should be consistent with the terminology.

Also, a [link to the example tests](https://github.com/rylnd/shpec/blob/master/shpec/shpec_shpec.sh#L71-L84) would be useful.

**Comment by rylnd** [comment]
@AdrieanKhisbe this is great!

The emacs mode is great, I've been wanting to write something similar for vim for a while now. Could you move that to a separate PR, though? It seems unrelated to the documentation, and we need to discuss where stuff like syntax files should live.

**Comment by AdrieanKhisbe** [comment]
Okey, I'll make a different pull request. Mode is very minimal, and I might add thinfs in the day to come.
Indeed that doesn't belong on the root, and we should find a name for the directory.

I already modified the readme in the new feature branch, pushing +mocking+ stubing after the matcher . (don't know if you had a look yet, I really want your input on this :)

File: README.md (hunk 1 above)
Comment (AdrieanKhisbe) [inline README.md]: I'll add it, (didn't know you could refer to specific line; That's great :) )

Comment (AdrieanKhisbe) [inline README.md]: What do you say about a trello board? :D

Comment (rylnd) [inline README.md]: I like trello, but we're already using Issues to track features and bugs. Is there a specific reason for wanting a trello board?

Comment (AdrieanKhisbe) [inline README.md]: We could Try waffle.io then. 
It was to discuss the feature I start to add, and keep track of the ideas, and one we should add now or then. (this is extra stuff, no issue ^^)

Comment (rylnd) [inline README.md]: If waffle.io allows everything we're doing to be visible through Github, I'm fine with trying that out. If not, I'd rather not add another tool; Issues with Labels and Milestones should be enough.

Comment (AdrieanKhisbe) [inline README.md]: Indeed everything is visible on github, just another view of the issues, milestone and co.
However you need to add me as a Contributor on the project so I can access to milestone, and issues. (I had a look on waffle, and you can only create.contribute to a repo where you are part of. So I only see my fork, so no issues)

**Comment by rylnd** [comment]
@AdrieanKhisbe once those comments are addressed, I'm good to merge this.

**Comment by rylnd** [comment]
@AdrieanKhisbe any updates on this PR? It's been open for more than a month with no progress.

**Comment by AdrieanKhisbe** [comment]
I forgot to close. I put the content of the emacs mode in the dedicated pull request.

if i remember well the documentation change are in the branch `dev` on which I started to prototype note feature. (I've been a bit busy these time, and I waiting to incorporate posix change to go on)

**Pull request closed without merging**